docs/examples/org-clojure-demo.org

```diff
@@ -0,0 +1,20 @@
+#+html_head_extra: <link rel="stylesheet" type="text/css" href="https://storage.googleapis.com/app.klipse.tech/css/codemirror.css" />
+
+* Sample Clojure code
+
+** sample defn
+
+#+BEGIN_SRC clojure
+(defn add [x y]  
+  (+ x y))
+#+END_SRC
+
+** sample invoking function
```

Comment:
Having Klipse integrated with org mode is  awesome!

Two things:
1. What kind of specific use cases do you have in mind?
2. There is a CSS conflict somewhere. In the Klipse snippets there is a light trail separating lines of code (see screenshot below).
![image](https://user-images.githubusercontent.com/955710/90356854-a305d100-e059-11ea-91c4-26fafcd960cb.png)